src/rak.js

```diff
@@ -4,18 +4,17 @@ const { waitFor } = require('./datatypes/util')
 
 let Client, Server, PacketPriority, EncapsulatedPacket, PacketReliability, Reliability
 
-module.exports = nativeRaknet => {
-  if (nativeRaknet) {
-    try {
-      ({ Client, Server, PacketPriority, PacketReliability } = require('raknet-native'))
-      return { RakServer: RakNativeServer, RakClient: RakNativeClient }
-    } catch (e) {
-      ({ Client, Server, EncapsulatedPacket, Reliability } = require('jsp-raknet'))
-      console.debug('[raknet] native not found, using js', e)
-      console.debug('You can suppress the error above by disabling `useNativeRaknet` in your options')
-    }
-  } else {
+module.exports = (backend) => {
+  if (!backend) backend = 'raknet-native'
```

Comment:
If this is set to ===false, it should use the JS implementation for backward compatibility

Reply:
The user may not set this option by default. When the user does not set this value, raknet-native is used by default.

Reply:
If the user does set this to false, `useNativeRaknet: false`, it should continue to use the JS implementation, not raknet-native

Reply:
please review again